src/librustc_session/lib.rs

```diff
@@ -1,10 +1,4 @@
 #![feature(crate_visibility_modifier)]
-#![feature(test)]
-
-// Use the test crate here so we depend on getopts through it. This allow tools to link to both
-// librustc_session and libtest.
-extern crate getopts;
-extern crate test as _;
```

Comment:
I'm wondering if this broke rustfmt and rls in https://github.com/rust-lang/rust/pull/71059#issuecomment-612676428

Reply:
Oh right link-related `extern crate`s still exist in rustc.

Reply:
The error:
```
error[E0464]: multiple matching crates for `getopts`
 --> /cargo/registry/src/github.com-1ecc6299db9ec823/rustc-ap-rustc_session-651.0.0/lib.rs:6:1
  |
6 | extern crate getopts;
  | ^^^^^^^^^^^^^^^^^^^^^
  |
  = note: candidates:
          crate `getopts`: /checkout/obj/build/x86_64-unknown-linux-gnu/stage1/lib/rustlib/x86_64-unknown-linux-gnu/lib/libgetopts-2a6246002b81ef60.rlib
          crate `getopts`: /checkout/obj/build/x86_64-unknown-linux-gnu/stage1/lib/rustlib/x86_64-unknown-linux-gnu/lib/libgetopts-c6adfde1f127313e.rlib

error[E0463]: can't find crate for `getopts`
 --> /cargo/registry/src/github.com-1ecc6299db9ec823/rustc-ap-rustc_session-651.0.0/lib.rs:6:1
  |
6 | extern crate getopts;
  | ^^^^^^^^^^^^^^^^^^^^^ can't find crate

error: aborting due to 2 previous errors
```

Reply:
Seems likely, but I'm pretty sure that once y'all bump the autopublished librustc_session crate then things should work. If not, please do let me know, we can re-add this here as needed.